### PR TITLE
Hamburger Menu

### PR DESCRIPTION
**Pull request by wajahatalli**
- Added HTML for mobile menu
- Added CSS to convert the hamburger bar to an "X" when pressed
- Javascript made this function, functional.
- Hamburger menu is now perfectly functional

### CODE REVIEW

File: query.css [hunk 1]
```diff
@@ -11,7 +11,7 @@
     background-color: white;
   }
 
-  .mobile_menu_icon {
+  .hamburger {
     display: none;
```
Comment (christianbiring1) [inline query.css:15]: - [ ] Kindly make sure your App is 100% responsive and fits small-size screens on the mobile version
- [ ] Kindly make sure your page fits the 100% width of the screen and not more. we do not need to scroll on the right to see the whole page's content.
Your design :point_down: 
image
image

File: style.css [hunk 2]
```diff
@@ -40,6 +41,96 @@ nav {
   color: #6070ff;
 }
 
+.hamburger {
+  z-index: 999;
+  position: relative;
+  display: block;
+  width: 40px;
+  cursor: pointer;
```
Comment (christianbiring1) [inline style.css:49]: - [ ] Kindly make sure you stick to the design as much as possible. the color, ....
In this case, you may consider making the background of the page when the hamburger menu is active as much blurry as it is in the Figma design 
Your design :point_down: 
image

Figma design :point_down: 
image

File: style.css [hunk 3]
```diff
@@ -40,6 +41,96 @@ nav {
...
+  appearance: none;
+  background: #fff;
+  outline: none;
+  border: none;
+}
+
```
Comment (christianbiring1) [inline style.css:55]: - [ ] Also, Kindly make sure all the body of the page is covered with the blue background when the hamburger is active,

YOur design :point_down: 
image

Figma design :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @wajahatalli , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] The Mobile menu is working as expected :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
`I HIGHLY RECCOMEND YOU IMPLEMENT THEM FOR THEY WILL TAKE YOUR PROJECT TO THE NEXT LEVEL` :rocket: 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: query.css (hunk 1 above)
Comment (wajahatalli) [reply query.css:15]: hi, 
its responsive on my side, a screenshot is attached.

Screenshot from 2022-11-16 14-56-35
Screenshot from 2022-11-16 14-56-28

**Pull request closed without merging**